### ISSUE TITLE
Option to install CP4D using IBM NFS (ibmc-file-gold-gid) storage class instead of Portworx is not present for IBM cloud

### ISSUE THREAD

**Issue by pragyasharma22** [IBM/cp4d-deployment#57]
We are using the provided scripts to automate provisioning and installation of CP4D on OC cluster. While reviewing the provided scripts, we can see only Portworx as storage call for IBM Cloud. Could you please let us know how we can use this script to use IBM NFS (ibmc-file-gold-gid) as storage class?

**Comment by shrumit** [comment]
Yes, unfortunately Portworx is the only storage option available using these Terraform scripts. You can install on ibmc-file-gold using the "Cloud Pak for Data" tile from the IBM Software catalog.

Also be advised that currently some ibmc-file-gold installations encounter a `access denied by server while mounting` error, if you encounter this please open a support ticket or post in the #conductors channel in Data and AI if you're an IBMer.

**Comment by parthakom2** [comment]
Closing this, please re-open if needed.
